proto/panacea/oracle/v2alpha2/query.proto

```diff
@@ -23,7 +23,12 @@ service Query {
 
   // OracleRegistration returns a OracleRegistration details.
   rpc OracleRegistration(QueryOracleRegistrationRequest) returns (QueryOracleRegistrationResponse) {
-    option (google.api.http).get = "/panacea/oracle/v2alpha2/oracle_registrations/unique_id/{unique_id}/addresses/{address}";
+    option (google.api.http).get = "/panacea/oracle/v2alpha2/oracle-registrations/{unique_id}/{address}";
```

Comment:
Unfortunately, cosmos-sdk use snake_case for rest API ([ref](https://github.com/cosmos/cosmos-sdk/blob/fc7ee0bf1e4e6afef4aed98515d418281edb7e62/proto/cosmos/bank/v1beta1/query.proto#L16)), so why don't we use the same style for the url?
```suggestion
    option (google.api.http).get = "/panacea/oracle/v2alpha2/oracle_registrations/{unique_id}/{address}";
```

Reply:
As far as I know, Rest API uses hyphen(-). Is there any reason to use underScore(_)?
It seems to use underScore in cosmos-sdk.

Reply:
I couldn't find any specific reason to use snake case for url. But I thought we would better to use the consistent style.